internal/client/apidocs/apidocs.go

```diff
@@ -150,8 +197,72 @@ func UpdateDocumentation(siteid string, id string, displayName string, openAPIDo
 	return nil, nil
 }
 
+// Export
+func Export(folder string) (err error) {
+	apiclient.ClientPrintHttpResponse.Set(false)
+	defer apiclient.ClientPrintHttpResponse.Set(apiclient.GetCmdPrintHttpResponseSetting())
+
+	siteids, err := sites.GetSiteIDs()
+	if err != nil {
+		return err
+	}
+
+	pageToken := ""
+	listdocs := listapidocs{}
+
+	for _, siteid := range siteids {
+		for {
+			l := listapidocs{}
+			listRespBytes, err := List(siteid, maxPageSize, pageToken)
+			if err != nil {
+				return fmt.Errorf("failed to fetch apidocs: %w", err)
+			}
+			err = json.Unmarshal(listRespBytes, &l)
+			if err != nil {
+				return fmt.Errorf("failed to unmarshall: %w", err)
+			}
+			listdocs.Data = append(listdocs.Data, l.Data...)
+			pageToken = l.NextPageToken
+			// write apidocs Documentation
+			for _, data := range l.Data {
+				respDocsBody, err := GetDocumentation(siteid, data.ID)
+				if err != nil {
+					return err
+				}
+				docFileName := fmt.Sprintf("apidocs_%s_%s.json", siteid, data.ID)
+				if err = apiclient.WriteByteArrayToFile(path.Join(folder, docFileName), false, respDocsBody); err != nil {
+					return err
+				}
+			}
+			if l.NextPageToken == "" {
+				break
+			}
+		}
+	}
+
+	respBody, err := json.Marshal(listdocs.Data)
+	if err != nil {
+		return err
+	}
+	respBody, _ = apiclient.PrettifyJSON(respBody)
+	return apiclient.WriteByteArrayToFile(path.Join(folder, "apidocs.json"), false, respBody)
+}
+
+func Import(conn int, folder string) (err error) {
+	entities, err := readAPIDocsFile(path.Join(folder, "apidocs.json"))
+	if err != nil {
+		clilog.Error.Println("Error reading file: ", err)
+		return err
+	}
+	return fmt.Errorf("not implemented")
```

Comment:
WIP?

Reply:
Yes. But now I have added most if not all the functionality.